### PR TITLE
add mac address section

### LINKED ISSUE #86: Resolve once and for all MAC address identifier questions
The MAC address is such an obvious identifier, particularly since it's so widely used. We've have multiple questions about it during reviews. We should probably add some simple text pointing that out.

E.g.

"The MAC address of a device is often used as an identity in existing implementations. A discussion of it has been omitted for brevity, but the MAC address could be used subject to the criteria in section 3.2"

### PR DESCRIPTION
**Pull request by klarose**
We've had lots of people asking about the mac address as an identifier.
Mention it, and point out that we won't give a deep discussion about it.

Fixes #86

### CODE REVIEW

**Comment by klarose** [comment]
This one will likely be controversial. I wouldn't mind input from a few others.

File: draft-ietf-capport-architecture.xml [hunk 1]
```diff
@@ -642,6 +642,14 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 to which the IP belongs.
           </t>
         </section>
+        <section anchor="id_example_mac_address">
+          <name>Media Access Control (MAC) Address</name>
+          <t>
+            The MAC address of a device is often used as an identity in existing implementations.
```
Comment (dcdolson) [inline draft-ietf-capport-architecture.xml:648]: I think change "identity" to "identifier".

File: draft-ietf-capport-architecture.xml [hunk 2]
```diff
@@ -642,6 +642,14 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 to which the IP belongs.
           </t>
         </section>
+        <section anchor="id_example_mac_address">
+          <name>Media Access Control (MAC) Address</name>
+          <t>
+            The MAC address of a device is often used as an identifier in existing implementations.
+            A discussion of it has been omitted for brevity, but the MAC address could be used
```
Comment (tfpauly) [inline draft-ietf-capport-architecture.xml:649]:
```suggestion
            This document does not discuss the use of MAC addresses within a captive portal system, but they can be used
```

**Comment by klarose** [comment]
Thanks for the reviews/suggestions.

**Pull request merged**